emcc.py

```diff
@@ -1222,7 +1245,7 @@ def get_bitcode_args(input_files):
     # First, generate LLVM bitcode. For each input file, we get base.o with bitcode
     for i, input_file in input_files:
       file_ending = filename_type_ending(input_file)
-      if file_ending.endswith(SOURCE_ENDINGS):
+      if file_ending.endswith(SOURCE_ENDINGS) or has_fixed_language_mode:
```

Comment:
i don't think this is right. what if the inputs are `no_suffix bitcode.bc`. Then only the first of those should hit this path, the one with a known non-source suffix should not be handled here.